src/config/routes.ts

```diff
@@ -1,23 +1,41 @@
-export const ROUTES = {
-  // Auth Stack
-  LOGIN: "Login",
-  MAIN_TABS: "MainTabs",
+export const ROOT_NAVIGATOR_ROUTES = {
+  AUTH_STACK: "AuthStack",
+  MAIN_TAB_STACK: "MainTabStack",
+} as const;
 
-  // Tab Stack
+export const MAIN_TAB_ROUTES = {
   TAB_HOME: "Home",
   TAB_SWAP: "Swap",
   TAB_HISTORY: "History",
   TAB_SETTINGS: "Settings",
 } as const;
 
+export const AUTH_STACK_ROUTES = {
+  WELCOME_SCREEN: "WelcomeScreen",
+  CHOOSE_PASSWORD_SCREEN: "ChoosePasswordScreen",
+  CONFIRM_PASSWORD_SCREEN: "ConfirmPasswordScreen",
+  RECOVERY_PHRASE_ALERT_SCREEN: "RecoveryPhraseAlertScreen",
+  RECOVERY_PHRASE_SCREEN: "RecoveryPhraseScreen",
+  IMPORT_WALLET_SCREEN: "ImportWalletScreen",
+} as const;
+
+export type AuthStackParamList = {
+  [AUTH_STACK_ROUTES.WELCOME_SCREEN]: undefined;
+  [AUTH_STACK_ROUTES.CHOOSE_PASSWORD_SCREEN]: undefined;
+  [AUTH_STACK_ROUTES.CONFIRM_PASSWORD_SCREEN]: undefined;
+  [AUTH_STACK_ROUTES.RECOVERY_PHRASE_ALERT_SCREEN]: undefined;
+  [AUTH_STACK_ROUTES.RECOVERY_PHRASE_SCREEN]: undefined;
+  [AUTH_STACK_ROUTES.IMPORT_WALLET_SCREEN]: undefined;
+};
+
 export type RootStackParamList = {
```

Comment:
Could we move `RootStackParamList` above `AuthStackParamList` to keep the same sequence pattern of the _ROUTES consts?

Reply:
Humm I think we could also move MAIN_TAB_ROUTES below AUTH_STACK_ROUTES, so we have it like this:

ROOT_NAVIGATOR_ROUTES

AUTH_STACK_ROUTES

MAIN_TAB_ROUTES

RootStackParamList

AuthStackParamList

MainTabStackParamList